docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+# Battery Monitor
+
+You can access the information related to the power consumption by applications or by hardware resources on a battery-powered device for a certain duration of time. The value is returned as the percentage of the total batttery capacity.
+
+The power consumption of an application is calculated by accumulating the usage of various individual resources used by the application.
+
+The Battery Monitor APIs provide following provisions to fetch the battery usage:
+
+- Getting battery usage information of a particular application.
+
+  You can [retrieve information of an application](#appusage_get) by specifying its application ID, [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+- Getting battery usage information of a particular resource.
+
+  You can [retrieve information of a single or muiltiple resources](#resourceusage_get) by specifying its [resource](#resource_key) IDs, and the time [duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the functions and data types of the Battery Monitor API (in [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) applications), include the `<battery_monitor.h>` header file in your application:
+
+   ```
+   #include <battery_monitor.h>
+   ```
+
+2. The application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+   </privileges>
+   ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Particular Application
+
+The battery usage information contains a combination of application ID, resource ID, and time duration.
+
+1. Fetching information of an application for a particular resource over a certain duration of time.
```

Comment:
As the 1st, 2nd, 3rd items of this list is not sequential, how about changing to unnumbered list?

Reply:
Yes, I have changed them to unnumbered list.